pkg/reconciler/ingress/controller.go

```diff
@@ -204,26 +203,24 @@ func NewController(ctx context.Context, cmw configmap.Watcher) *controller.Impl
 	}
 
 	// Get the current list of ingresses that are ready and seed the Envoy config with them.
-	ingressesToSync, err := getReadyIngresses(ctx, knativeClient.NetworkingV1alpha1())
+	ingressesToSync, err := getReadyIngresses(ingressInformer.Lister())
 	if err != nil {
 		logger.Fatalw("Failed to fetch ready ingresses", zap.Error(err))
 	}
 	logger.Infof("Priming the config with %d ingresses", len(ingressesToSync))
 
-	// The startup translator uses clients instead of listeners to correctly list all
-	// resources at startup.
 	startupTranslator := generator.NewIngressTranslator(
 		func(ns, name string) (*corev1.Secret, error) {
-			return kubernetesClient.CoreV1().Secrets(ns).Get(ctx, name, metav1.GetOptions{})
+			return secretInformer.Lister().Secrets(ns).Get(name)
```

Comment:
Just to make sure, we are now using the filtered secretInformer instead of the unfiltered kubeclient. It seems that Kourier can have just the secret filtered here: https://github.com/knative-sandbox/net-kourier/blob/main/pkg/reconciler/informerfiltering/util.go#L44. So this should be fine, right?

Reply:
Yes, I think no problem. 

Secrets for internal-encryption has `networking.internal.knative.dev/certificate-uid` label in serving repo and Secrets for DomainMapping BYO should be added `networking.internal.knative.dev/certificate-uid` by users so we just need to take care of the filtered secrets when users enabled the filtering.